packages/cli/src/zosfiles/ZosFilesBase.handler.ts

```diff
@@ -39,33 +39,31 @@ export abstract class ZosFilesBaseHandler implements ICommandHandler {
      *
      * @returns {Promise<void>}
      */
-    public async process(commandParameters: IHandlerParameters) {
+    public async process(commandParameters: IHandlerParameters): Promise<void> {
         const sessCfg: ISession = ZosmfSession.createSessCfgFromArgs(
             commandParameters.arguments
         );
         const sessCfgWithCreds = await ConnectionPropsForSessCfg.addPropsOrPrompt<ISession>(
-            sessCfg, commandParameters.arguments, {parms: commandParameters}
+            sessCfg, commandParameters.arguments, { parms: commandParameters }
         );
-
         const session = new Session(sessCfgWithCreds);
-        const response = await this.processWithSession(commandParameters, session);
-
         commandParameters.response.progress.endBar(); // end any progress bars
-        // Print out the response
-        if (response.commandResponse) {
-            commandParameters.response.console.log(response.commandResponse);
-        }
-
-        // Return as an object when using --response-format-json
-        commandParameters.response.data.setObj(response);
-
-        // Ensure error gets thrown if request was unsuccessful.
-        // Sometimes it is useful to delay throwing an error until the end of the handler is
-        // reached, for example the upload API needs to return an API response even when it fails.
-        if (!response.success && response.commandResponse) {
-            throw new ImperativeError({
-                msg: response.errorMessage || response.commandResponse
-            });
+        try {
+            const response = await this.processWithSession(commandParameters, session);
```

Comment:
I believe a few unit tests might be failing because of the `response.success === false` possibility.

Reply:
my unit tests arent failing, im confused about what you mean

Reply:
@ATorrise Tests are failing in GH workflows, here's an example run: https://github.com/zowe/zowe-cli/actions/runs/10944131154/job/30385249824